### PR TITLE
Rename syl6eqssr to eqsstrrdi

### PR DESCRIPTION
**Pull request by jkingdon**
This is in iset.mm and set.mm

### CODE REVIEW

**Review by avekens — APPROVED** [review]
approved after merge conflicts are solved

**Comment by jkingdon** [comment]
Only change was a simple conflict resolution. Planning on merging when tests pass.

**Pull request merged**
